drivers/client_location_history/config/routes.rb

```diff
@@ -9,6 +9,9 @@
     resources :clients, only: [:none] do
       get :map, on: :member
     end
+    resources :projects, only: [:none] do
```

Comment:
This is a blind copy/paste, I don't really know what `only: [:none]` is doing.

Reply:
`only: [:none]` is there because we wanted to reuse the resources defined in the main routes file, but not stomp on them (none could be anything that isn't in the main routes file.)  If you don't have sub pages defined, you wouldn't need this.  I think you want something like:
```
resources :projects, only: [:none] do
  get :map, on: :member
end
```